libs/@local/hash-graph-sdk/node/src/entity.ts

```diff
@@ -0,0 +1,14 @@
+import type { Entity as GraphApiEntity } from "@local/hash-graph-client/api";
+import type { EntityRecordId } from "@local/hash-subgraph";
+
+export class Entity {
+  #entity: GraphApiEntity;
```

Comment:
Is `#entity` preferred over `private entity`?

Reply:
Yes, it provides runtime privacy, `private` does not